.github/scripts/deploy.sh

```diff
@@ -18,32 +18,36 @@ validate_create() {
     exit 1
   fi
 }
+
+# Extract tag from GitHub ref
+TAG=${GITHUB_REF#refs/tags/}
```

Comment:
@cdoyle-temple In the gencon-solr build.sh file we are using $RELEASE_TAG  and here you are using TAG=${GITHUB_REF#refs/tags/}.  Can we try to stay consistent and pick one way of doing things?  My main goal with this work is to be consistent across projects.